buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -51,28 +51,28 @@ object Repositories {
 
 object Libraries {
     object Versions {
-        const val kotlin = "1.6.21"
-        const val coroutines = "1.6.1-native-mt"
+        const val kotlin = "1.7.10"
+        const val coroutines = "1.6.4"
         const val jetpack = "1.1.0"
         const val constraintLayout = "1.1.3"
-        const val ktx = "1.6.0"
+        const val ktx = "1.8.0"
         const val material = "1.5.0"
         const val pinEditText = "1.2.3"
         const val sqlLiteJdbc = "3.36.0"
         const val desugaring = "1.1.5"
         const val workManager = "2.7.1"
-        const val fragment = "1.2.5"
+        const val fragment = "1.5.1"
         const val compose = "1.2.0-beta01"
-        const val composeMaterial = compose
+        const val composeMaterial = "1.2.1"
         const val composeMaterial3 = "1.0.0-alpha11"
         const val composeActivity = "1.4.0"
-        const val composeNavigation = "2.4.2"
+        const val composeNavigation = "2.5.0"
         const val accompanist = "0.24.7-alpha" // https://github.com/google/accompanist/issues/1162
         const val composeConstraint = "1.0.0"
-        const val hilt = "2.38.1"
+        const val hilt = "2.43.2"
```

Comment:
Maybe we need also to upgrade hilt plugin version ? ( `com.google.dagger:hilt-android-gradle-plugin` inside `build.gradle.kts` in `buildSrc` module)